### PR TITLE
Fixing link focus

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
1. Removing display content from link
2. Removing extra margin at the top of link
3. Adding before link

## Related Tickets & Documents
https://github.com/forem/forem/issues/9585

## QA Instructions, Screenshots, Recordings

https://user-images.githubusercontent.com/8092120/105316092-63ce4400-5be6-11eb-8453-4108eaaa3a78.mp4

## Added tests?

- [ ] Yes
- [x] No, and this is why: This is a css change
- [ ] I need help with writing tests

## Added to documentation?

- [ ] [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
- [ ] README
- [x] No documentation needed

## What gif best describes this PR or how it makes you feel?

Happy

### CODE REVIEW

File: app/assets/stylesheets/preact/onboarding-modal.scss [hunk 1]
```diff
@@ -606,9 +606,6 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
   }
 
   label {
-    @extend .crayons-field;
-    @extend .crayons-field__label;
```
Comment (Rafi993) [inline app/assets/stylesheets/preact/onboarding-modal.scss:610]: This was adding `margin-top` to the a tag.

File: app/assets/stylesheets/preact/onboarding-modal.scss [hunk 2]
```diff
@@ -650,10 +647,6 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
         margin-bottom: 0;
       }
 
-      a {
-        display: contents;
```
Comment (Rafi993) [inline app/assets/stylesheets/preact/onboarding-modal.scss:654]: This was preventing focus on the link using keyboard when pressing tab.

File: app/javascript/onboarding/components/IntroSlide.jsx [hunk 3]
```diff
@@ -144,7 +144,7 @@ class IntroSlide extends Component {
                         checked={checked_code_of_conduct}
                         onChange={this.handleChange}
                       />
-                      You agree to uphold our{' '}
+                      You agree to uphold our&nbsp;
```
Comment (Rafi993) [inline app/javascript/onboarding/components/IntroSlide.jsx:147]: Empty string `{'  '}` was not rendering so I instead replaced it with `&nbsp;`

File: app/assets/stylesheets/preact/onboarding-modal.scss (hunk 1 above)
Comment (aitchiss) [reply app/assets/stylesheets/preact/onboarding-modal.scss:610]: I see the problem you mention with the margin-top, but removing these `@extend`s alters the sizing a little, and loses the link with our design system scss

If you reinstate these lines, you should be able to get the same effect while keeping the other styles with

```scss
    @extend .crayons-field;
    @extend .crayons-field__label;

    a {
      margin-top: 0;
    }
```

**Review by aitchiss — CHANGES_REQUESTED** [review]
This looks great - thanks for the improvements here! Just one tiny note on the label scss if you're able to tweak 👍

File: app/assets/stylesheets/preact/onboarding-modal.scss (hunk 1 above)
Comment (Rafi993) [reply app/assets/stylesheets/preact/onboarding-modal.scss:610]: I have made the above changes now.

**Review by aitchiss — APPROVED** [review]
Thanks @Rafi993 - looks great and works perfectly 👍

**Review by katiedavis — APPROVED** [review]

**Pull request merged**